docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx

```diff
@@ -26,6 +26,9 @@ following information about each flight is given:
 You may need to enable the functionality to upload a CSV or Excel file to your database. The following section
 explains how to enable this functionality for the examples database.
 
+**Note:** To enable `Upload Excel`, you need to make sure that the `xlrd` dependency is installed,
+which you can execute `pip install apache-superset[excel]` to install.
+
```

Comment:
While you're at it, would you mind updating line 26 to state that we also support The Parquet columnar format for data uploads?

Reply:
@villebro  Of course, it has been updated.